WoWsShipBuilder.ViewModels/Helper/ShipComparisonDataWrapper.cs

```diff
@@ -0,0 +1,340 @@
+
+    //Concealment
+    public decimal ConcealmentFromShipsBase { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentBySea;
+    public decimal ConcealmentFromShipsOnFire { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentBySeaFire;
+    public decimal ConcealmentFromShipsSmokeFiring { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentBySeaFiringSmoke;
+    public decimal ConcealmentFromPlanesBase { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentByAir;
+    public decimal ConcealmentFromPlanesOnFire { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentByAirFire;
+    public decimal ConcealmentFromSubsPeriscopeDepth { get; } = ShipDataContainer.ConcealmentDataContainer.ConcealmentBySubPeriscope;
+
+    //Survivability
+    public decimal SurvivabilityShipHp { get; } = ShipDataContainer.SurvivabilityDataContainer.HitPoints;
+    public decimal SurvivabilityFireMaxAmount { get; } = ShipDataContainer.SurvivabilityDataContainer.FireAmount;
+    public decimal SurvivabilityFireDuration { get; } = ShipDataContainer.SurvivabilityDataContainer.FireDuration;
+    public decimal SurvivabilityFireDps { get; } = ShipDataContainer.SurvivabilityDataContainer.FireDPS;
+    public decimal SurvivabilityFireMaxDmg { get; } = ShipDataContainer.SurvivabilityDataContainer.FireTotalDamage;
+    public decimal SurvivabilityFireChanceReduction { get; } = ShipDataContainer.SurvivabilityDataContainer.FireReduction;
+    public decimal SurvivabilityFloodingMaxAmount { get; } = ShipDataContainer.SurvivabilityDataContainer.FloodAmount;
+    public decimal SurvivabilityFloodingDuration { get; } = ShipDataContainer.SurvivabilityDataContainer.FloodDuration;
+    public decimal SurvivabilityFloodingDps { get; } = ShipDataContainer.SurvivabilityDataContainer.FloodDPS;
+    public decimal SurvivabilityFloodingMaxDmg { get; } = ShipDataContainer.SurvivabilityDataContainer.FloodTotalDamage;
+    public decimal SurvivabilityFloodingTorpedoProtection { get; } = ShipDataContainer.SurvivabilityDataContainer.FloodTorpedoProtection;
+
+    //Sonar
+    public decimal? SonarReloadTime { get; } = ShipDataContainer.PingerGunDataContainer?.Reload;
+    public decimal? SonarTraverseSpeed { get; } = ShipDataContainer.PingerGunDataContainer?.TraverseSpeed;
+    public decimal? SonarTurnTime { get; } = ShipDataContainer.PingerGunDataContainer?.TurnTime;
+    public decimal? SonarRange { get; } = ShipDataContainer.PingerGunDataContainer?.Range;
+    public decimal? SonarWidth { get; } = ShipDataContainer.PingerGunDataContainer?.PingWidth;
+    public decimal? SonarSpeed { get; } = ShipDataContainer.PingerGunDataContainer?.PingSpeed;
+    public decimal? SonarFirstPingDuration { get; } = ShipDataContainer.PingerGunDataContainer?.FirstPingDuration;
+    public decimal? SonarSecondPingDuration { get; } = ShipDataContainer.PingerGunDataContainer?.SecondPingDuration;
+
+    //RocketPlanes
```

Comment:
Use a helper collection so you need to do the filtering only once. Ignore the memory overhead of doing so.